core/src/main/scala/tofu/syntax/foption.scala

```diff
@@ -0,0 +1,23 @@
+package tofu.syntax
+
+import cats.Monad
+import cats.syntax.applicative._
+import cats.syntax.flatMap._
+import cats.syntax.functor._
```

Comment:
For a pure performance reason, I would recommend using `tofu.syntax.monadic._` instead of these three